ui/src/app/kvm/components/StorageResources/StorageMeter/StorageMeter.tsx

```diff
@@ -1,25 +1,24 @@
 import KVMResourceMeter from "app/kvm/components/KVMResourceMeter";
 import StoragePopover from "app/kvm/components/StorageColumn/StoragePopover";
-import type {
-  KVMStoragePoolResource,
-  KVMStoragePoolResources,
-} from "app/kvm/types";
-import { calcFreePoolStorage } from "app/kvm/utils";
+import type { KVMStoragePoolResources } from "app/kvm/types";
+import { calcFreePoolStorage, getSortedPoolsArray } from "app/kvm/utils";
+import type { Pod } from "app/store/pod/types";
 
 type Props = {
+  defaultPoolId?: Pod["default_storage_pool"];
   pools: KVMStoragePoolResources;
 };
 
-const StorageMeter = ({ pools }: Props): JSX.Element | null => {
-  const poolsArray = Object.entries<KVMStoragePoolResource>(pools);
-  if (poolsArray.length !== 1) {
+const StorageMeter = ({ defaultPoolId, pools }: Props): JSX.Element | null => {
+  const sortedPools = getSortedPoolsArray(pools);
+  if (sortedPools.length !== 1) {
     return null;
   }
-  const [, pool] = poolsArray[0];
+  const [, pool] = sortedPools[0];
```

Comment:
I know this wasn't added in this branch, but do you remember why we just pick the first pool in the array here? If so, it could be worth leaving as a comment.